api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -127,6 +127,18 @@ public static String plainBootstrapAddress(String clusterName) {
         return bootstrapServiceName(clusterName) + ":9092";
     }
 
+    /**
+     * Returns the address (<em>&lt;host&gt;</em>.<em>&lt;namespace&gt;</em>.svc:<em>&lt;port&gt;</em>)
+     * of the internal plain bootstrap {@code Service} for a {@code Kafka} cluster of the given name and namespace.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @param namespace  The {@code metadata.namespace} of the {@code Kafka} resource.
+     * @return The address of the corresponding bootstrap {@code Service}.
+     * @see #tlsBootstrapAddress(String)
+     */
+    public static String namespacedPlainBootstrapAddress(String clusterName, String namespace) {
```

Comment:
This should not be in the `api` module. It is just your assumption that plain listener might be on port 9092. But there is no such rule and it might not apply for users. So this might apply to the STs, but it should be in some ST utility class. Not in the `api` which is public artifact and should be generally applicable.